addons/website/__manifest__.py

```diff
@@ -101,4 +100,334 @@
+            # after //link[last()]
+            'website/static/src/scss/options/colors/user_theme_color_palette.scss',
+        ],
+        '_assets_secondary_variables': [
+            # before //link
+            ('append', 'website/static/src/scss/secondary_variables.scss'),
+        ],
+        'assets_tests': [
+            # inside .
+            'website/static/tests/tours/reset_password.js',
+            # inside .
+            'website/static/tests/tours/rte.js',
+            # inside .
+            'website/static/tests/tours/html_editor.js',
+            # inside .
+            'website/static/tests/tours/restricted_editor.js',
+            # inside .
+            'website/static/tests/tours/dashboard_tour.js',
+            # inside .
+            'website/static/tests/tours/specific_website_editor.js',
+            # inside .
+            'website/static/tests/tours/public_user_editor.js',
+            # inside .
+            'website/static/tests/tours/website_navbar_menu.js',
+            # inside .
+            'website/static/tests/tours/snippet_version.js',
+            # inside .
+            'website/static/tests/tours/website_style_edition.js',
+        ],
+        'assets_backend': [
+            # after //link[last()]
+            'website/static/src/scss/website.backend.scss',
+            # after //link[last()]
+            'website/static/src/scss/website_visitor_views.scss',
+            # after //link[last()]
+            'website/static/src/scss/website.theme_install.scss',
+            # after //script[last()]
+            'website/static/src/js/backend/button.js',
+            # after //script[last()]
+            'website/static/src/js/backend/dashboard.js',
+            # after //script[last()]
+            'website/static/src/js/backend/res_config_settings.js',
+            # after //script[last()]
+            'website/static/src/js/widget_iframe.js',
+            # after //script[last()]
+            'website/static/src/js/theme_preview_kanban.js',
+            # after //script[last()]
+            'website/static/src/js/theme_preview_form.js',
+        ],
+        'qunit_suite': [
+            # after //script[last()]
+            'website/static/tests/dashboard_tests.js',
+            # after //script[last()]
+            'website/static/tests/website_tests.js',
+        ],
+        '_assets_frontend_helpers': [
+            # before //link
+            ('append', 'website/static/src/scss/bootstrap_overridden.scss'),
+        ],
+        '_assets_frontend_helpers_user_custom': [
+            # before //link
+            ('append', 'website/static/src/scss/user_custom_bootstrap_overridden.scss'),
+        ],
+        'assets_frontend': [
+            # replace //script[@src='/web/static/src/js/public/public_root_instance.js']
+            ('replace', '/web/static/src/js/public/public_root_instance.js', 'website/static/src/js/content/website_root_instance.js'),
```

Comment:
You can use the same syntax as the other the globs when specifying a target, so in this case `'web/static/src/js/public/public_root_instance.js'`. I find it more consistent.